### ISSUE TITLE
Saving found bugs

### ISSUE THREAD

**Issue by mgayanov** [crytic/echidna#921]
Can you please tell where Echidna saves crashes?

For example, i run

`echidna-test --corpus-dir=./corpus tests/solidity/basic/flags.sol`

fuzzer finds bug, but where it are stored?

**Comment by ggrieco-tob** [comment]
They are stored in the reproducers folder, when `--corpus-dir` is enabled.

**Comment by mgayanov** [comment]
> They are stored in the reproducers folder, when `--corpus-dir` is enabled.

Thank you for reply

I run test
`echidna-test --corpus-dir=./corpus tests/solidity/basic/flags.sol`

And I have this
```
echidna_sometimesfalse: failed!💥  
  Call sequence:
    set0(0)
    set1(16730805812149397316656615579029322054136400570)

Event sequence: Flag(false) from: Test@0x00a329c0648769A73afAc7F9381E08FB43dBEA72, Flag(false) from: Test@0x00a329c0648769A73afAc7F9381E08FB43dBEA72
echidna_alwaystrue:  passed! 🎉
echidna_revert_always:  passed! 🎉

Unique instructions: 343
Unique codehashes: 1
Corpus size: 2
Seed: 6630367978650638640
```

I cannot find `Call sequence` in `./corpus` dir, I have:

```
./corpus
  coverage/
    -3311684445542986203.txt
    1997679056438704053.txt
  covered.1674638898.html
  covered.1674638898.txt
```

`grep -r 16730805812149397316656615579029322054136400570 ./corpus` returns nothing

Thanks

**Comment by ggrieco-tob** [comment]
Which echidna version are you using?

**Comment by mgayanov** [comment]
> Which echidna version are you using?

Echidna 2.0.4

**Comment by ggrieco-tob** [comment]
It was implemented in Echidna 2.0.5. Please upgrade and you will see the new feature.

**Comment by mgayanov** [comment]
> It was implemented in Echidna 2.0.5. Please upgrade and you will see the new feature.

Thank you, it works!
